live-examples/js-examples/proxyhandler-apply.html

```diff
@@ -0,0 +1,14 @@
+<pre>
+<code id="static-js">var proxy1 = new Proxy(function() {}, {
+  apply: function(target, thisArg, argumentsList) {
+    console.log('called: ' + argumentsList.join(', '));
+    // expected output: "called: 1, 2, 3"
+
+    return argumentsList[0] + argumentsList[1] + argumentsList[2];
+  }
+});
+
+console.log(proxy1(1, 2, 3));
+// expected output: 6
```

Comment:
Yes this is new to me also Will. Your example is more practical I think. As long as it's fine to introduce other concepts with `Reflect.apply(...arguments)` which is neater but potentially a source of confusion for the user due to the fact we're illustrating another type of `apply`.

We could instead use something like `argumentsList[0] + argumentsList[1]` - it's clearly not as neat and less flexible so I can see the merits to both approaches.

Reply:
I'm reading more about using `Using Reflection with Proxies` https://developers.google.com/web/updates/2016/02/es2015-proxies#using_reflection_with_proxies so now thinking that your original example could be fine @wbamberg

Reply:
Yes, I think it's OK to use Reflect with these examples, because they're quite closely related, and if we avoid Reflect in cases like these, then we are not teaching good practices.